openmdao/api.py

```diff
@@ -105,6 +105,8 @@
     OMInvalidCheckDerivativesOptionsWarning
 
 
+from openmdao.utils.general_utils import wing_dbg
+
```

Comment:
Are we keeping this permanently in API?  I guess it's ok, and it is easier to just set the env var now.

Reply:
Yeah I just put this in the api file so you and I could activate wing debugging with a single line, and I don't know about you but I often want to switch back and forth between wing debugging and just dumping stuff to stdout with prints, so it's easier to just precede my run command with WING_DBG=1 or not to control whether wing takes over or not.  It's not a big deal though and I can take it out of the api file if people don't think it should be in there.